curriculum/challenges/english/25-front-end-development/review-html-accessibility/671a87a39b88245a579c2271.md

```diff
@@ -9,7 +9,46 @@ dashedName: review-html-accessibility
+- **Importance for inputs to have an associated label**: You should use the `<label>` element to associate labels with form inputs. This helps screen readers and other assistive technologies understand the purpose of the input.
+- **Importance of good `alt` text**: You should use the `alt` attribute to provide a text alternative for images. This helps screen readers and other assistive technologies understand the content of the image.
+- **Importance of good link text**: You should use descriptive link text to help users understand the purpose of the link. This helps screen readers and other assistive technologies understand the content of the link.
+- **Best practices for making audio and video content accessible**: You should provide captions and transcripts for audio and video content to make it accessible to people with hearing impairments. You should also provide audio descriptions for video content to make it accessible to people with visual impairments.
+- **`tabindex` attribute**: used to specify the order in which elements should receive focus when using the tab key to navigate the website. This can help users with mobility impairments navigate the website more easily.
+- **`accesskey` attribute**: used to define a keyboard shortcut for an element. This can help users with mobility impairments navigate the website more easily.
+
+## WAI-ARIA, Roles, and Attributes
+
+- **WAI-ARIA**: a set of attributes that can be added to HTML elements to improve accessibility. It provides additional information to assistive technologies about the purpose and structure of the content.
+- **Aria roles**: a set of predefined roles that can be added to HTML elements to define their purpose. This helps screen readers and other assistive technologies understand the content of the website. Examples include `role="button"`, `role="dialog"`, and `role="navigation"`.
+- **`aria-label` and `aria-labelledby` attributes**: The `aria-label` attribute can be used to provide a text alternative for an element. The `aria-labelledby` attribute can be used to associate an element with another element that serves as its label. This helps screen readers and other assistive technologies understand the purpose of the element.
+- **`aria-hidden` attribute**: The `aria-hidden` attribute can be used to hide an element from screen readers. This can be useful for decorative elements that do not provide any meaningful content.
+- **`aria-expanded` attribute**: used to indicate whether an element is expanded or collapsed. This helps screen readers and other assistive technologies understand the state of the element.
+- **`aria-live` attribute**: used to indicate that an element's content is live and should be announced by screen readers. This is useful for elements that are updated dynamically.
```

Comment:
```suggestion
- **`aria-live` attribute**: Used to indicate that an element's content is important enough to require that any changes to the content should be announced immediately to screen reader users. This can include status messages like updating the number of results returned from a search, or an error message displayed after an unsuccessful form submission.
```

Just being picky here. I didn't like that `aria-live` was being defined with the word "live".